src/ui/component/stepper/stepperProgress/progressBar/progressBar.tsx

```diff
@@ -0,0 +1,20 @@
+import classNames from 'classnames'
+import type { FC } from 'react'
+import type { StepStatus } from '../stepperProgress'
+import './progressBar.scss'
+
+type ProgressBarState = 'active' | Extract<StepStatus, 'complete' | 'incomplete'>
```

Comment:
I can't see clearly the advantage of using here an `Extract` utility type over as example a `Pick` one, in terms of intention.

Reply:
Sorry, I didn't notice the type of the step statuses! 
LGTM !